presto-spi/src/main/java/com/facebook/presto/spi/security/AccessDeniedException.java

```diff
@@ -228,14 +228,14 @@ public static void denyRevokeTablePrivilege(String privilege, String tableName,
         throw new AccessDeniedException(format("Cannot revoke privilege %s on table %s%s", privilege, tableName, formatExtraInfo(extraInfo)));
     }
 
-    public static void denyShowGrants(String user)
+    public static void denyShowGrants()
     {
-        denyShowGrants(user, null);
+        denyShowGrants(null);
     }
 
-    public static void denyShowGrants(String user, String extraInfo)
+    public static void denyShowGrants(String extraInfo)
```

Comment:
At present, `SHOW GRANTS` doesn't have any provision for showing grants for users other than the current user. So the user is implicitly the current session user. If and when we add the ability to show grants for other users, we could user user information.